seneca.js

```diff
@@ -2074,6 +2078,18 @@ function make_seneca (initial_options) {
     return this
   }
 
+  // Inspired by https://github.com/hapijs/hapi/blob/master/lib/plugin.js decorate
+  function api_decorate (property, method) {
+    assert.ok(property, 'property must be specified')
```

Comment:
If you want to shave off a few characters, you can drop the `.ok` just do `assert(property, 'property must be specified')`